app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessPublicTransportForm.java

```diff
@@ -21,6 +20,27 @@
 
 	private void setTitle()
 	{
-		setTitle(R.string.quest_wheelchairAccess_public_transport_title);
+		String highway = null;
+		OsmElement element = getOsmElement();
+		if (highway == null) {highway = element != null && element.getTags() != null ? element.getTags().get("highway") : null;}
+		if (highway == null) {highway = element != null && element.getTags() != null ? element.getTags().get("railway") : null;}
+		if (highway == null) {highway = element != null && element.getTags() != null ? element.getTags().get("public_transport") : null;}
```

Comment:
I think neither element nor element.getTags() can be null at this point. If element would be null, to which element would the quest apply to, then? Also, if the element had no tags, why was a quest created for it based on the tags given in the quest?